### PR TITLE
Update swagger.yaml

### PR DESCRIPTION
**Pull request by garyluu**
This updates the swagger.yaml to have the new refresh by org endpoints and also the security definition and endpoint authorization.   This was generated using the generateYaml test.  Not sure why there's so many formatting changes.

### CODE REVIEW

**Comment by denis-yuen** [comment]
Re-opening for testing

File: dockstore-integration-testing/src/test/java/io/dockstore/client/cli/SystemClientIT.java [hunk 1]
```diff
@@ -97,9 +100,11 @@ private static ApiClient getWebClient(boolean correctUser, boolean admin) throws
         File configFile = FileUtils.getFile("src", "test", "resources", "config");
         INIConfiguration parseConfig = Utilities.parseConfig(configFile.getAbsolutePath());
         ApiClient client = new ApiClient();
+        ApiKeyAuth bearer = (ApiKeyAuth)client.getAuthentication("BEARER");
+        bearer.setApiKeyPrefix("BEARER");
+        bearer.setApiKey((correctUser ? parseConfig
+            .getString(admin ? Constants.WEBSERVICE_TOKEN_USER_1 : Constants.WEBSERVICE_TOKEN_USER_2) : "foobar"));
```
Comment (garyluu) [inline dockstore-integration-testing/src/test/java/io/dockstore/client/cli/SystemClientIT.java]: These 4 lines are mostly likely not needed.

Comment (denis-yuen) [reply dockstore-integration-testing/src/test/java/io/dockstore/client/cli/SystemClientIT.java]: Nope, in this suite 10 tests pass, 12 fail without this.

Comment (garyluu) [reply dockstore-integration-testing/src/test/java/io/dockstore/client/cli/SystemClientIT.java]: I just check this branch out and it passed.

Comment (denis-yuen) [reply dockstore-integration-testing/src/test/java/io/dockstore/client/cli/SystemClientIT.java]: I just tried this out as well for the above comment. 
Are you sure you're running integration tests?

**Pull request merged**